### ISSUE TITLE
[Suggestion]Add BacterialVat quest.

### ISSUE THREAD

**Issue by yukieiji** [GTNewHorizons/GT-New-Horizons-Modpack#6115]
#### Which modpack version are you using?
2.0.9.0Beta2
#
#### What do you suggest instead/what changes do you propose?
Since Bacterial Vat is now needed to produce BioMedium, I believe a quest with instructions on how to use it, etc. is needed.
In fact, I don't even know how to make it work at all.

**Comment by bartimaeusnek** [comment]
Step1: Build it.
Step2: Throw Items, Fluids and if nessecary radiation material in it. ( You will need a Petri dish with a culture, which would need a seperate Questline aswell)
Step3: keep output hatch at HALF FILLED LEVEL for optimal efficiency
Step4: ???
Step5: Profit

**Comment by chochem** [comment]
yeah, as bart says, a questline for the BioLab would be great too.

It would be great if the quest for the Vat has some more explanations then what bart wrote above, like how precise is this *half filled* requirement, or what exactly means *optimal efficiency*. Was it x1000 (or x100?) for input+output fluid while keeping solids + recipe time fixed?? I am not sure, I havent used/tested it quite yet.

Some more info on the radio hatch would be great too. E.g. it only takes rods/long rods. Is there an overview of Sievert values?

**Comment by chochem** [comment]
ok, I got some of the info for the quest from bot. Lets just collect a few pieces of info that need to be in the vat quest:

- keep output hatch at HALF FILLED LEVEL for optimal efficiency (how precise?)
- optimal efficiency means: 1000x input and output fluid while keeping solids + recipe time fixed.
- Sievert values are just the number of the element (whith nq+ at 140 and nq* at 150)

**Comment by Prometheus0000** [comment]
- What about the biovat stuff? It seems really complicated with the DNA samples and plasmids and the different component/circuit/whatever they were called
- Presumably you would use a large output hatch and turn off the machine when it's below X amount in the hatch, possibly with a fluid regulator?
- Does it need to be EXACTLY half filled, I thought it was half or more?
- Does the multiplier scale, or is it just 1000x all at once?
- Recommend hatches for in/output for the 1000xing, so it can actually hold that much. Or was it like 1L at a time, I forget?
- Recommend a method of keeping it at half-filled that's been tested to work.
- What recipes are useful, and what are required, and what unlocks when.
- Is there a difference in normal/long rods? Time, radiation amount?
- Mention the glass tiers, and what requires what especially if there are followup quest for different fluids
-Is there enough BW stuff for it's own tab? There's the windmill, pump, mega x3, CAL and bio/bact vat, but I'm not sure that's enough. Do the Ross planets+all the weird ore deserve an info quest?

**Comment by yukieiji** [comment]
How about quest like this one?
The details of each quest are not written down because of my lack of English...

![2020-05-21_14 02 05](https://user-images.githubusercontent.com/40425798/82525608-c01b2880-9b6c-11ea-8c5c-a16cbc0a75d8.png)
![2020-05-21_14 02 13](https://user-images.githubusercontent.com/40425798/82525615-c3161900-9b6c-11ea-8adf-ab139fb52cb4.png)
![2020-05-21_14 02 55](https://user-images.githubusercontent.com/40425798/82525617-c3161900-9b6c-11ea-95ae-163728473e5e.png)
![2020-05-21_14 02 59](https://user-images.githubusercontent.com/40425798/82525618-c3aeaf80-9b6c-11ea-942c-06d75c6e4cf4.png)
![2020-05-21_14 06 17](https://user-images.githubusercontent.com/40425798/82525619-c3aeaf80-9b6c-11ea-8692-c510d2786a3d.png)

**Comment by Regian24** [comment]
Nice but it needs a description.

If I reached the bio vat stage i could write them down, but I'm only HV lololol.

**Comment by yukieiji** [comment]
I've written a detailed description of each quest, How about it?

・Bacterial Vat
Bacterial Vat is a unique multiblock machine.
It's not required in the tier where this quest was opened, but there are many useful recipes and you'll need in later tiers, so you should make them.
This machine will require Glass-Tier that are used for 2 and 3 layers and a creature.
Glass-Tier is calculated as LV as Tier 1, which is the same as the voltage. The glass required by this quest will be recognized as Tier 4.
Creature can be made by BioLab and put in the controller block.

・Sievert for Bacteria
Some Vat recipes require for Sievert, i.e. radiation. so attach this Radio Hatch to your Vat and let the radioactive material in.
Radio Hatch can insert rod of radioactive material and accumulate that, and the item inserted determines the Sievert value. Accumulated items will disappear with the passage of time.
Sievert value and accumulation and disappearance time can be checked with a portable scanner.
For example, if you insert Uranium rod you get with this quest reward, Sievert is 92 and the amounts 1, disappear at 1588t/79s/1m per 1kg.

・Clean Biohazard
Need clean and IV Bio Lab to create a creature to make a cheap circuit for LuV, so let's create one.
Also, since you'll be using Raw Glowth Medium in large amount, it's a good idea to made it now.

・Make Vat more efficient!!
Looking at some Vat's recipe you are likely discouraged by too little Fluid to consume and too little Fluid to be produced.
Don't worry, Fluid consumption and production shown on the NEI are the lowest efficiency.
The efficiency of Vat depends on the amount of Fluid in Output Hatch.
Maximum efficiency is reached when more than half of Output Hatch is filled with Fluid, and the amount of Fluid consumed and produced is multiplied by 1000.

**Comment by chochem** [comment]
I went over it a bit, what do you think?

I also did some tests: 1) the efficiency increase is gradual. I dont know what the formula is, but its fairly forgiving with pretty high efficiency already in the 30%-70% range. 2) it goes down again beyond 50% as expected from barts statement. 

---

・Bacterial Vat
The Bacterial Vat is a unique multiblock machine.
The Vat is not mandatory in this tier, but there are many useful recipes and you'll need it later on, so let's build one!
Most recipes required a minimum glass tier, a culture, and possibly some radioactive material. 

The glass tier is calculated with tier 1 for LV, so this quest requires you to make tier 4 glass. The culture is made in a BioLab and put in the controller, while the radioactive material goes into a radio hatch.

・Sievert for Bacteria
Some Vat recipes have a Sievert requirement. So attach this Radio Hatch to your Vat to input radioactive material!
The Radio Hatch accepts rods of radioactive material and accumulates that. The inserted item determines the Sievert value. Accumulated items will disappear with the passage of time.
Sievert value, accumulation, and disappearance time can be checked with a portable scanner. The Sievert value is given by the atomic number of the material.

For example, if you insert Uranium rod you get with this quest reward, the Sievert value is 92, the amount 1, and it disappears after 1588t/79s/1m per 1kg.

・Lets do some Microbiology!
You need an IV Bio Lab in a Cleanroom to create a culture needed for wetware circuits, so let's make one. Since you'll be using Raw Growth Medium in large amounts, it's a good idea to start now!

・Make the Vat more efficient!!
Looking at some of the recipes for the Vat you are likely discouraged by the small amount of fluid being produced per recipe.
Don't worry! Fluid consumption and production shown in NEI are just the base line for the lowest efficiency. The efficiency of the Vat depends on the amount of fluid in the output hatch. The efficiency gradually increases as the output hatch fills up and reaches maximum efficiency when it is half-filled. If the hatch fills even more, the efficiency goes down again, so try to keep it close to half!
At maximum efficiency the amount of fluid consumed and produced per recipe is multiplied by 1000 while everything else stays the same.

**Comment by Regian24** [comment]
Just a little fix, let's make it so that it says that naquadah is 140 and naquadria is 150
----------------------------------------------------------------
Sievert for Bacteria
Some Vat recipes have a Sievert requirement. So attach this Radio Hatch to your Vat to input radioactive material!
The Radio Hatch accepts rods of radioactive material and accumulates that. The inserted item determines the Sievert value. Accumulated items will disappear with the passage of time.
Sievert value, accumulation, and disappearance time can be checked with a portable scanner. The Sievert value is given by the atomic number of the material. 
Enriched Naquadah is worth 140 Sievert and Naquadria is 150.

**Comment by yukieiji** [comment]
@Regian24 I don't think it's necessary to write Naquadah and Naquadria, which are not available at the time this quest is opened, but why should they be?

**Comment by chochem** [comment]
looks like you included some of my fixes but not others, which is fair. But you got to at least fix these:

'You need clean and the IV Bio Lab to create a creature'
'you need clean' is not something you can say, and its a culture, not a creature.

Also, 'clean biohazard' sounds very strange. maybe better 'Biohazard in the Cleanroom!'. 

Unless I am not getting some references here of course :P

**Comment by Prometheus0000** [comment]
- Why would you make it _four_ separate quests? There's no reason to do that.
- They should be _two_ quests, one for the biolab, one for the bacterial vat.
- There should be followup quests for the different cultures/fluids (I'm not suggesting you make those)
- You should also include all possible rods/long rods (Can you use them? Did you test it?) in the quest, how long they last, and the sievert values. There shouldn't be that many after all.
- For 'Make the Vat more efficient' did you test and see what the efficiency values are while using a fluid detector? Did you try other methods? Or search for some on discord? Why did you not include some values in the quest?
- Why did you make the quests so fast when #6144 may happen? Now they'll need to be changed right away.
- And what about the actual recipes? I don't see any in the quests above?
- What happened to listing useful recipes? Or what ones are needed?
- And why unlocked by tier 5? Isn't that EV? Aren't they available at HV?
- And why is there a link from the diode quest of all things?
- And what happened to explaining how the biolab worked? You can't just make a quest that says 'make it' and call it a day, you have to explain ALL of the mechanics of the device. I don't see any mention of plasmids, DNA samples, petri dishes, or _any_ of the things you would need to explain, like how to make cultures to begin with?
- Why are you telling people to make the IV bio lab? It starts at HV?
- And why the f are you telling people to make wetware/raw growth medium at EV? Isn't that ZPM/UV? That should be a followup quest at _best._
- Why in the 'Bacterial Vat' quest do you tell people to make tier 4 glass? That isn't necessary at all to start using it? That's like saying you should make the chemical plant out of EV machine hulls when you first make it? Aren't most of the recipes tier 0?
- Why use so many '!'? It looks really stupid, and isn't good english.
- Your english (typos/grammer) itself is a problem, but that can be cleaned up later.
- Why is the input bus EV? They don't do anything above HV, you know?
- And why are the input and output hatches EV? They should be LV so you can complete the quest easily and then thrown into a chest and then use the ones you actually want to use. Not everyone will want to use the same tier after all.
- Why is the radio hatch EV? What's the difference between the different tiers? Is there any?
- How do you find the atomic number of the material? Is it the number on the item (U-238 for example). You need to tell people this!
- Why would you have the 'Make the Vat more efficient' quest use an output _bus?_ And why ULV? And if it's supposed to be a hatch, why different from the previous quest when it's supposed to be used in the same multiblock? And the smaller the hatch the harder it is get the efficiency right, but easier to start. But you didn't mention any of that.
- Did you add coins properly to the other quests?
- Probably other stuff I forgot.

I know properly researching the devices is a pain, which is why I was putting it off.

**Comment by Dream-Master** [comment]
@Prometheus0000 come down i want to check the quests and adjust it ;)

**Comment by Prometheus0000** [comment]
Didn't you already check it before you merged the PR? It just seems incredibly sloppy work, you know? You shouldn't complete the quests and then go 'but how do I _use_ it'? And then have to go on to discord to find out. The quests should explain these mechanics if possible.

**Comment by chochem** [comment]
just a few to hopefully help out a bit in making it better:

> How do you find the atomic number of the material? Is it the number on the item (U-238 for example). You need to tell people this!

no, that is not the number. atomic number is just the protons. there is no in game way to see it afaik. check wikipedia I suppose.


> And what happened to explaining how the biolab worked? You can't just make a quest that says 'make it' and call it a day, you have to explain ALL of the mechanics of the device. I don't see any mention of plasmids, DNA samples, petri dishes, or any of the things you would need to explain, like how to make cultures to begin with?

Making cultures in the biolab is very self-explanatory, you just follow NEI. plasmids, DNA samples, etc are a different story. I have no idea what those do. are they needed for anything? you have to ask bart there.

> For 'Make the Vat more efficient' did you test and see what the efficiency values are while using a fluid detector? Did you try other methods? Or search for some on discord? Why did you not include some values in the quest?

I actually did some tests. but they are all at strange random numbers, like 20.4% efficiency at 5.3% fill level. not with a fluid detector(?), just by actually looking at the amount of fluid in the hatch. I am not sure what you mean with 'methods' here.

> Why is the input bus EV? They don't do anything above HV, you know?
And why are the input and output hatches EV? They should be LV so you can complete the quest easily and then thrown into a chest and then use the ones you actually want to use. Not everyone will want to use the same tier after all.

agreed

> Why are you telling people to make the IV bio lab? It starts at HV?

The IV Biolab is needed for wetwares. but if the quest is tiered pre-IV due to other recipes then it should be lower of course.

> Why in the 'Bacterial Vat' quest do you tell people to make tier 4 glass? That isn't necessary at all to start using it? That's like saying you should make the chemical plant out of EV machine hulls when you first make it? Aren't most of the recipes tier 0?

most of the *important* recipes have a glass tier. wetwares need T5 actually. not sure what is best to ask for here. depends on where the quests is tiered. If its in HV I would still ask for the borosilicate glass blocks -even though others work- because these can be upgraded to the later tiers.

> They should be two quests, one for the biolab, one for the bacterial vat.

radio hatch is not needed for everything, so giving that an extra quest looks good to me. The fourth one could be a pure checkbox quest about the efficiency ideally or just integrated into the main one. Asking for a fluid detector cover and ulv output bus (hatch?) is probably not good. there are many ways to achieve the efficiency, no need to force the player down one specific path.

**Comment by Prometheus0000** [comment]
By methods, I meant methods of keeping it half-filled, like your PA canner shenanigans (though I don't fully understand how that works). Like, SFM is apparently broken for BW fluids (will it be fixed?) but can you use OC? Just how much efficiency can you get with a fluid detector on a ULV hatch vs LV/MV/HV/etc. And if the output hatch changes, how to regulate putting stuff back in?

I think that it should start at the earliest possible point you can make/use them, with followup quests about bio medium, mysterious crystal and other useful liquids. You can add necessary quests in-between (radio hatch, higher glass tiers, etc.) But those should be followup quests, not where you start.

But that seems like a lot of not-mulitblock stuff for a multiblock tab which is why I suggested it might be time for a BW tab before.

**Comment by botn365** [comment]
PA canner is not to keep it halsf full but is to fill the hatch with cells instead of runing the bio lab

**Comment by chochem** [comment]
> like your PA canner shenanigans (though I don't fully understand how that works).

thats for filling from a tank, not for a keeping a fill level.

but yeah you can certainly do multiple different things here! I use fluid detector+pipe+machine controller. always have 99.8+ efficiency except for start-up. but thats just one option, so maybe dont push the player either way.

**Comment by yukieiji** [comment]
@Prometheus0000 

> And why unlocked by tier 5? Isn't that EV? Aren't they available at HV?

The reason I unlocked the Bacterial Vat quest at T5 is because the EV circuit and EV Tier glass were required in the Vat recipe. Since the EV circuit can only be made after the late HV, I thought that T5 would not be a problem.

> Why is the input bus EV? They don't do anything above HV, you know?
And why are the input and output hatches EV? They should be LV so you can complete the quest easily and then thrown into a chest and then use the ones you actually want to use. Not everyone will want to use the same tier after all.
Why in the 'Bacterial Vat' quest do you tell people to make tier 4 glass? That isn't necessary at all to start using it? That's like saying you should make the chemical plant out of EV machine hulls when you first make it? Aren't most of the recipes tier 0?
Why is the radio hatch EV? What's the difference between the different tiers? Is there any?

I requested the hatch/bus/glass/Radio Hatch that to match the tier being opened in the quest, should I not specify the hatch and bus?

> There should be followup quests for the different cultures/fluids (I'm not suggesting you make those)
And what happened to explaining how the biolab worked? You can't just make a quest that says 'make it' and call it a day, you have to explain ALL of the mechanics of the device. I don't see any mention of plasmids, DNA samples, petri dishes, or any of the things you would need to explain, like how to make cultures to begin with?
Why are you telling people to make the IV bio lab? It starts at HV?

Sure, I thought I'd need a followup quest for creatures and such, but I decided that should be done in a separate quest tab(such as new BW tab).

> And why the f are you telling people to make wetware/raw growth medium at EV? Isn't that ZPM/UV? That should be a followup quest at best.

The Clean Lab's quest will be opened in IV.
I think don't have a problem starting GrowthMedium at that time. because it would be a good help rather than a problem.

> You should also include all possible rods/long rods (Can you use them? Did you test it?) in the quest, how long they last, and the sievert values. There shouldn't be that many after all.

I tested it, and it was the same except for Amount.

> Why would you have the 'Make the Vat more efficient' quest use an output bus? And why ULV? And if it's supposed to be a hatch, why different from the previous quest when it's supposed to be used in the same multiblock? And the smaller the hatch the harder it is get the efficiency right, but easier to start. But you didn't mention any of that.
And why is there a link from the diode quest of all things?

It is my mistake

**Comment by Prometheus0000** [comment]
I have done some research.
- Radio hatch tier controls how many rods/long rods you can put in at a time. 1kg = 1 rod = 1 amount. Long rod = 2kg = 2 amount. You don't have space, so you can't put a long rod in an HV hatch. This maxes at UV (no recipes for higher tiers), which can hold 6kg. But it's constantly depleting even when not running, so this might not be a good idea unless you run it constantly.
- It will use any rods/long rods you put in (that work with it) and fill it's internal buffer, so putting a stack into a UV one will consume 6 rods, while an HV will only consume 1. Unless you can send exactly the right amount, you should expect wastage (again, unless you run it constantly). And auto-filling it will just waste all your materials.

- I checked some rods:
U-235 92sv 80s
U-238 92sv 80s
Pu-239 94sv 66s
Pu-241 94sv 66s
Nq 130sv 14.5
Nq+ 140sv 10.5s
Nq* 150sv 8s
There are 3 recipes that need 83 sv, which indicates I might have missed a rod?

There are others that have a lower amount like Naq. Alloy, B. Plutonium, Duranium (why?) and some others, but I'm not sure they're even supposed to have a value. The 'radioactive' GT++ ones also seem to work, but I don't wanna check all the values of all those~. Some are really high like LiBeF...etc which is 222, or radioactivemineralmix (unobtainable?) which is 548. But the highest required is 150, and it seems it gets used up faster if it has a higher Sv value, so I doubt these are worth investigating anyway.

- You can use the screwdriver to access the 'Radiation Shutter Control', so you can set the 'containment level' whatever that means? I saw that the bar up top reduces when you increase the number, so it _probably_ emits less sv, but it doesn't seem to reduce the rate at which the item is used up, making it seem kind of pointless? Wouldn't you just use a lesser rod instead? Or is it planned to require sv ranges in the future?

**Comment by Dream-Master** [comment]
@Prometheus0000 it might be Bismuth

**Comment by Prometheus0000** [comment]
- I checked the recipe and it does require EV circuits and EV glass, but both of these are made in HV, therefore it's an HV machine. Same for the biolab. And they both use HV hulls too. I'd also say that EV circuits is more mid-HV, since I did the rocket stuff _months_ after the EV circuits, while still being in HV.
- The thing about specifying bus/hatch tiers is that not everyone will want to use the same thing. I might want to use all LV, someone else might want to use IV, and another use different sizes for all of them. This is why the GT++ quests don't specify them for example. I agree with chochem that it would probably be best to specify the HV borosilicate glass so you can upgrade it later. We can then either have followup quests for higher glass tiers, and then quests linked off of them, or tell people what glass tier they need in the specific 'make this liquid' type quests instead. Only about 21/91 require more than T0 after all. Though most of the recipes do seem kinda useless at glass T0.
- I still don't understand why you would waste resources on making raw growth medium in IV? What are you even making?
- I agree with chochem that the fourth quest should be folded into the first. You should also add the info above about radio hatches into the third.
- Another important thing to note is that recipe lookup doesn't work properly for the bacterial vat right now. For example if you check uses of Agar, you'll just see an LCR recipe, but if you find a recipe that uses it, Raw Growth Medium, you can see it in the recipe and follow it back. This obviously makes things harder to figure out, and should be mentioned, unless it's fixed soon.

**Comment by Prometheus0000** [comment]
I checked, and Bismuth rods are indeed 83 sv and 198s

**Comment by yukieiji** [comment]
Changed EV to HV and not require hatches, the fourth Quest to a checkbox.
The integration of the first and fourth quests wasn't done because they were too long.

The second Quest is a followup Quest that preparation for LuV(wetware) and Lab in need of a cleanroom. If there's another followup quest coming up, might as well integrate it into that one.
Which info should be added to the third quest? I think the info about which materials have what sievert value and how long they disappear should be put together in separate a quest.

**Comment by Prometheus0000** [comment]
About the plasmids, you can look at #5049 for more details, but the important parts that are difficult to figure out from NEI are recipes 83 and 92.

First off is duplicating cultures with 92, but get a Data Orb with the proper culture you want copied first from one of the other recipes:
![1](https://user-images.githubusercontent.com/8517039/82749338-887fcc80-9d76-11ea-9266-0e6d983ad40e.png)
![2](https://user-images.githubusercontent.com/8517039/82749340-8c135380-9d76-11ea-9730-68c59621bf53.png)

83 is about changing one culture into another (or maybe using a plasmid to make it from a sterilized one? I didn't test it):
![3](https://user-images.githubusercontent.com/8517039/82749444-499e4680-9d77-11ea-92ec-0a37f3cea268.png)
![4](https://user-images.githubusercontent.com/8517039/82749447-4c00a080-9d77-11ea-9662-f36c8d738f08.png)
![5](https://user-images.githubusercontent.com/8517039/82749450-4efb9100-9d77-11ea-8497-4f2794c9eba3.png)

There are also recipes to make DNA Samples from Cultures, and a followup recipe to get the Data Orb version from the DNA Samples. Then a recipe to use Plasmid Sample Flasks along with 2 different Data Orbs (though I don't know why, are some only made this way?) to make a Plasmid Sample Flask with the type specified, which you use in recipe 83.

You also need to use the correct module of course.

**Comment by yukieiji** [comment]
I certainly feel that quest is necessary as well, but I don't think they are quest to implement in the MultiBlock Goal tab...

**Comment by Prometheus0000** [comment]
I believe if you change Minecraft's GUI Scale option you would have much more space to see the text. This is under ESC -> Options -> Video Settings -> GUI Scale. Change it from Auto to something else, like normal.

The upper half of my comment about radio hatches needs to go in the quest about the radio hatch. Where else would it go? A second quest about it would be weird.

Of course some of my stuff was about testing, and can be summarized too. Basically the main point is the rods stats (use the full names though)
Bi 83sv 198s
U-235 92sv 80s
U-238 92sv 80s
Pu-239 94sv 66s
Pu-241 94sv 66s
Nq 130sv 14.5
Nq+ 140sv 10.5s
Nq* 150sv 8s

You say the second quest is a followup quest, but it's before the first one? Or does it have the requirement of the cleanroom _and_ bacterial vat? That would be weird though, since it's the bacterial vat that requires the bio lab, and the bio lab only _sometimes_ requires the cleanroom. And you need an HV one listed if it opens up in HV. A wetware quest should require you to make the fluid, not the IV bio lab. And what if people want to make the LuV/ZPM/UV one instead? Not that I'm sure what tier you could make before making bio medium, but you should be able to make the LuV one at least. Specifying the tier of a machine should be done as little as possible to give players more freedom to choose what they want to make. Honestly I'd like to be able to specify 'make any bio lab' (or hatch or bus or whatever) but I'm not sure that kind of thing is possible.

**Comment by Prometheus0000** [comment]
I'm not sure people need to know about plasmids at this point wither, but I wanted to write down what I figured out for later at least.

**Comment by yukieiji** [comment]
Opening the second quest is a prerequisite for IV and BacterialVat.
In order to make GrowthMedium for making wetware you need to make a creature using a BioLab of IV or higher in Cleanroom. It's a followup quest to that.

Added description of items disappearing even if Machine not active

**Comment by Prometheus0000** [comment]
I don't think you mean that you need it to get to IV, but like I said, the bacterial vat is a machine you can make at HV, thus it is an HV machine. The accompanying bio lab also has an HV version, and an explanation of how to use it should also be HV. Or are you suggesting you wait until IV so you can get an explanation of an HV machine? So that you can use it to make a material for ZPM or so? So that you can, I dunno, stockpile it without using it? If you want to do that that's fine, but you shouldn't expect others to want to. All the useful recipes are LuV+? Or is there one that the NEI not working isn't showing me?

**Comment by yukieiji** [comment]
I don't know what you mean.
The description of the creature that can be made in Bio Lab is written in the Bacterial Vat quest, but is it not enough?
The second quest to show that it's easier to make GrowthMedium in IV, and it shouldn't be worth more than that.

**Comment by Prometheus0000** [comment]
I'm saying the questline should go bio lab(2) -> bacterial vat(1) -> radio hatch(3) and the efficiency one(4).
With an explanation of how to use them in the first two.

After that should be bacterial vat -> higher glass tier quest -> recipes that are useful that need that tier of glass.

Also at least 1 quest for explaining the plasmids and such linked from the bio lab quest.

Though it might get messy depending on how many there are, and I'm not expecting you to do them or for them to be on this tab. That can be done later.

You don't need to be so attached to making it be about raw growth medium, the recipe is broken and can't even be done right now anyway (just tried, see https://github.com/GTNewHorizons/GT5-Unofficial/pull/288).

**Comment by yukieiji** [comment]
I feel that the instructions on how to use Bio Lab deviate from the content of the MultiBlock Goal tab. and this issue is Bacterial Vat's quests, not Bio Lab's quests...
I think the current second quest is necessary because it is an induction to make the creature needed to make make wetware circuit raw material in vat.

**Comment by Prometheus0000** [comment]
The bio lab can't be separated from the bacterial vat though, it's not optional or something. And you didn't put making it into the bacterial vat quest either. Where else should it be explained, but in a quest to get a bio lab? And if we're talking about things that 'deviate from the content of the MultiBlock Goal tab' wetware and growth medium would be even more so? Which is why I suggested a BW tab. But I'm waiting on #6144 since I don't want to rewrite things. There's no reason to be in a rush you know?

**Comment by yukieiji** [comment]
Because the Bio Lab quest requires an explanation of plasmids and DNA other than creature creation. I'm saying that if you make them a questline, you're deviating from the MultiBlock Goal tab
As you can see in the second sentence, I'm making second quest as an induction for the recipe that is required to be made using Vat. If a large number of people don't need it, I'll remove it.

Is there a need for a quest 'The Clean Lab' as an induction to wetware(Raw Growth Medium)?

**Comment by Prometheus0000** [comment]
All these quests are going to be moved later (probably) so it doesn't matter too much. A raw growth medium quest should have a requirement of the cleanroom, but the IV bio lab and the HV bio lab don't require it.

**Comment by yukieiji** [comment]
I created a new quest-line in BartWork and migrated the quests, How about quest like this one?
![2020-05-24_21 26 39](https://user-images.githubusercontent.com/40425798/82754056-5cf7f300-9e05-11ea-98ba-92749598e3dd.png)

**Comment by bartimaeusnek** [comment]
> 
> 
> I created a new quest-line in BartWork and migrated the quests, How about quest like this one?
> ![2020-05-24_21 26 39](https://user-images.githubusercontent.com/40425798/82754056-5cf7f300-9e05-11ea-98ba-92749598e3dd.png)

please dont. most quests are in the individual tabs anyways (i.e. Platinum overhaul, circuit programmer, etc.)

**Comment by yukieiji** [comment]
All right, I'll put it back.

**Comment by Prometheus0000** [comment]
Then we should move the biolab/bacterial vat quests to the HV tab, and then add further quests linked from x-tier glass on the other tabs later on. Though I'm not sure where to put ZPM+ voltage fluids in that case. Crystal is made in LuV and wetware ZPM right? So there's no place to put a wetware quest yet since there's no ZPM tab.

**Comment by chochem** [comment]
yeah, you can make the wetwares in ZPM, except for the wetware mainframe which is in UV.

I think a good idea for right now could be to turn the LuV tab into a LuV+ tab.
like: Tier 7 - 32768+ EU (LuV+)

If that tab becomes full at some point, then it can be split into LuV and ZPM+

**Comment by Prometheus0000** [comment]
That's probably the best solution for now. I still need to look at how many useful fluids there are, I don't want to add quests for mediocre-level usefulness ones.

**Comment by chochem** [comment]
from what I can tell sofar:

mandatory:
- one of: bacteria, bacterial sludge, mutagen. imo bacteria is the clear winner.
- raw growth medium
- raw bio medium
- seaweedBroth (I think, not there yet)

also important but not strictly mandatory from what I can tell:
- mutagen (mostly for gendustry, also an option for raw crystal chips)

maybe some usefulness:
- bacterial sludge as one of the options for raw crystals and to kickstart mutagen production
- any of the bacteria types that can be turned into liquid dna for environmenal processor (gendustry)
- xenoxene for some ultra late game stuff

**Comment by Prometheus0000** [comment]
- Also a way to get bacteria. You can make it in the vat or get it from growth medium (liquid) which you get from the solid version.
- Fulvic Acid can become oil, but this seems pointless considering pumping
- Maybe the general purpose one?

**Comment by yukieiji** [comment]
I think bacterial vat is a multiblock machine, so it's not an HV tab (although Chemical Plant is in the HV tab...)
How about this?

- HV tab:First Bio Lab Quest(Usage and Leading to Vat, DNA, Plasmid, etc.)
- IV tab:The Clean Lab
- MultiBlock Goal tab:Bacterial Vat、Sievert for Bacteria、Make the Vat more efficient!、Glass-TierQuest(such as EBF Coil)

**Comment by Prometheus0000** [comment]
Actually it's probably the CP which should be moved to the MB tab now that I think about it. I didn't see any other MB machines when I looked, it's the CP that confused me. Moron didn't even make any quests for the fluids it makes since it was in flux so much (and it's changing _again)._

I agree with points 1 and 3, but I still don't think you should be making raw growth medium at IV, what would you use it for? The only thing you can do is heat it up to make the sterilized version. After that you:
1) Make wetware boards (but this is an LuV circuit assembler recipe)
2) Make neuro processing units (if you had a zpm assline)
3) Make stemcells and bacterial sludge (but only if you have unknown crystal shards/mysterious crystals, which is T5 rocket at minimum, which is LuV/ZPM)

There's no reason for it to be an IV quest, it should be ZPM. You can also get bacterial sludge from the bacterial vat directly, and you get stemcells from chicken eggs (check the LuV tab). So there's no reason to overclock your LCR for those.

You should put it on the LuV tab (as per chochem's suggestion) linked from crystalprocessor mainframe next to the egg one. And it shouldn't require you to get an IV lab, since you can choose if you want that or a LuV/ZPM one for yourself. You can suggest using the IV one though. The quest completion requirement should be the correct culture, and then a cell of raw growth medium as a second page.

If you're going to add in the text for plasmids and DNA samples, make sure you test it, I copped out because it seemed annoying to check. But feel free to give a summary and leave the detailed explanation for later instead.

**Comment by Regian24** [comment]
why not create a ZPM and UV tab?

**Comment by Prometheus0000** [comment]
Because I have no idea what to put in it. Same for other quest makers. I don't think moron has even gotten past IV, and I don't know if dream even plays. I intend to do it when I get there, but expect that to take a long time. It's better not to have a tab than give inaccurate info, you can go to the discord after all. And I think there's up to UHV or UEV machines too? But I'd have to figure out the tectech stuff, what ores you need to get to progress, what machines are needed to progress, etc. Plus as I understand it there aren't really fancy new mechanics for the most part so it's not needed immediately or anything.

**Comment by chochem** [comment]
I agree that a zpm or uv tab only makes sense if there is more quests.
Part of the problem is, that many of the quests are currently elsewhere (fusion, tectech, legendary quest lines, gendustry stuff in how to bee, ...)

However I can happily give you a write up of zpm and the missing parts of LuV and what I think should get quests if you think that helps.

**Comment by Regian24** [comment]
I meant, even if there are like 3 quests, we can still add to it in the future

**Comment by Prometheus0000** [comment]
Well, if you want to write that up in another ticket I guess I could do that. I was waiting until I went through the legendary stuff since I can't even tell what tier that stuff is or whether it's even important (and I've been taking a break recently).

For example, what tier is the tectech stuff? LuV? ZPM? It unlocks from crystalprocessor, but it seems to be about making the researcher for UHV/UEV stuff. But the UHV circuit you can research there might be ZPM, but I'm not sure. And is there other tectech stuff you need to make for which there's (of course) no documentation?

The Legendary Quests tab seems to start at IV since you need a T4 rocket and you can make that in IV, but the second last quest requires ZPM stuff?

But it seems to unlock the next tab, which is full of weird quests, including one that requires a UHV machine hull of all things, and only seems to get you the ability to get draconic cores, which I don't know if they tie into the progression anywhere.

Also, I figured dream would have to give permission for a new tab, but maybe I'm overthinking that.

**Comment by Regian24** [comment]
legendary quests starts at HV, you need a Tier1 Rocket for the Moon dungeon blocks

**Comment by Prometheus0000** [comment]
Yeah, but I meant to complete the first quest, which requires blocks from Io.

**Comment by Regian24** [comment]
> Yeah, but I meant to complete the first quest, which requires blocks from Io. I don't think there's any other tab where the tier of the quest changes as you progress. Except the magic ones I guess.

yeh I see

**Comment by chochem** [comment]
I will make a longer write-up for somewhere else. just to answer some questions:
the tectech tab is pretty clearly UV, you need 4A UV energy hatches. In turn it is first needed for the wetware mainframe (first UHV circuit)

the legendary quests tab is aimed at ZPM (quite literally, a ZPM is the reward). But you should be able to complete it in late LuV. (needs T4 rocket to start and needs T5 rocket to complete)

the legendary tier X quests need a T8 rocket to start. I have not done those yet. somewhere late UV-UHV I guess. it is the only path to draconic cores in this pack which an important crafting ingredient (e.g. for wyvern/draconic/infinity armor)

**Comment by Regian24** [comment]
> the legendary tier X quests need a T8 rocket to start. I have not done those yet. somewhere late UV-UHV I guess. it is the only path to draconic cores in this pack which an important crafting ingredient (e.g. for wyvern/draconic/infinity armor)

i think it's kind of stupid that a quest is the only way to get a draconic core in this pack. they should add a crafting recipe

**Comment by chochem** [comment]
I think the questline is actually pretty cool. the quest for the draconic cores is repeatable if thats what you are worried about. pretty much like a crafting recipe BUT with the main difference that you have to go through all the quests one time and do like bloodmagic and thaumcraft stuff, etc.

**Comment by Regian24** [comment]
> I think the questline is actually pretty cool. the quest for the draconic cores is repeatable if thats what you are worried about. pretty much like a crafting recipe BUT with the main difference that you have to go through all the quests one time and do like bloodmagic and thaumcraft stuff, etc.

i agree, but only if you need just a few draconic cores. if somewhere at some point you'd need 200 i wouldn't want to do that quest 200 times

**Comment by Prometheus0000** [comment]
See #5144 #5157. Also in the PR dream asked them to put it in EMT, but didn't so it stalled out.

**Comment by Regian24** [comment]
#5157 is actually a really good idea lol

**Comment by yukieiji** [comment]
@Prometheus0000 Are you saying it's useless to make a raw growth medium from IV and save it up? I don't feel that it's particularly wasteful.
And I think it's strange to put IV tier of machine quests in the quest tab of LuV....

**Comment by Prometheus0000** [comment]
I'm saying it's weird to expect people to want to do that. I didn't save up galena or sphalerite in LV/MV so I could use it to make indium in IV, I mined some more. You're free to do that, but making a quest in IV, and telling other people to save up for something _hundreds of hours away_ for it is weird. The eggs for stem cells for example can be made in HV, but they're on the LuV tab because there's no use for them until then. 

About your other point, I just said that it shouldn't require an IV machine.

**Comment by yukieiji** [comment]
Are you aware that what is written in the quest is presented as a single idea, not a compulsion? Tell them that hoarding makes it easier, and it's up to the individual to follow or not follow, right?
And they present it in the sub quests instead of the main quest. That means you don't necessarily have to look at it, clear it, or follow it.

**Comment by Prometheus0000** [comment]
I cleaned up the text, but the DNA samples and such need to be explained better in a followup quest.
